examples/kprobe/bpf/kprobe_example.c

```diff
@@ -0,0 +1,25 @@
+#include "../../../testdata/common.h"
```

Comment:
Maybe we should just ship a `vmlinux.h` + `bpf_helper_defs.h` in the examples folder? Something like https://github.com/cloudflare/rakelimit/tree/master/include/bpf basically.